doc/man/cockpit.conf.xml

```diff
@@ -71,6 +71,19 @@ Origins = https://somedomain1.com https://somedomain2.com:9090
           </informalexample>
         </listitem>
       </varlistentry>
+	    <varlistentry>
+	      <term><option>Shell</option></term>
```

Comment:
Hmmm, I feel like this should be an undocumented option for now. WDYT? It's just not something admin/user configurable. It's really a delivery option.

Which reminds me, do we need to have multiple locations for cockpit.conf, where we have a defaults file? I think even in a container we may want to allow an deployer to theoretically customize the cockpit.conf in /etc. Anyway that would be a separate pull request.